go.mod

```
@@ -1,56 +1,59 @@
 module github.com/rancher/wharfie
 
-go 1.24.2
+go 1.24.0
+
+toolchain go1.24.7
```

We should avoid introducing the `toolchain`. Just match the go version used in v1.34.1-k3s1, aka `1.24.6` 

```suggestion
go 1.24.6
```